utils/menu_plugin.py

```diff
@@ -0,0 +1,92 @@
+        ('rh_courses', 'Courses')
+    )
+}
+
+files_path = data_path.join("menu_plugin")
+vmdb_path = local("/var/www/miq/vmdb")
+
+
+def menu_plugin_setup():
+    # Copying the files from menu_plugin folder to their respective locations
+    with store.current_appliance.ssh_client as ssh:
+        # default module section
+        status, result = ssh.run_command(
+            "find {} -type d -ls | grep 'menubar'".format(vmdb_path.join("product")))
+        if bool(status):
+            ssh.run_command("mkdir /var/www/miq/vmdb/product/menubar")
+        ssh.put_file(files_path.join("/custom_redhat_section.yml").strpath,
+            vmdb_path.join("/product/menubar/custom_redhat_section.yml").strpath)
+        ssh.put_file(files_path.join("/custom_redhat.yml").strpath,
+            vmdb_path.join("/product/menubar/custom_redhat.yml").strpath)
+        ssh.put_file(files_path.join("/custom_redhat_courses.yml").strpath,
+            vmdb_path.join("/product/menubar/custom_redhat_courses.yml").strpath)
+        status, result = ssh.run_command(
+            "find {} -type d -ls | grep 'miq_product_features'".format(vmdb_path.join(
+                "db/fixtures")))
+        if bool(status):
+            ssh.run_command("mkdir /var/www/miq/vmdb/db/fixtures/miq_product_features")
+        ssh.put_file(files_path.join("/redhat-miq_product_features.yml").strpath,
+            vmdb_path.join(
+                "/db/fixtures/miq_product_features/redhat-miq_product_features.yml").strpath)
+        # Restarting the evmserverd to reflect the changes
+        store.current_appliance.restart_evm_service(rude=True)
+    menu_plugin_module_link_setup()
+    sel.force_navigate("dashboard")
+
+
+def menu_plugin_module_link_setup():
+    new_branch = menu.branch_convert(sections)
+    menu.nav.add_branch('toplevel', new_branch)
+
+
+def menu_plugin_teardown():
+    with store.current_appliance.ssh_client as ssh:
+        # default module section
+        ssh.run_command("rm -rf {}".format(vmdb_path.join(
+            "/product/menubar/custom_redhat_section.yml").strpath))
+        ssh.run_command("rm -rf {}".format(vmdb_path.join(
+            "/product/menubar/custom_redhat.yml").strpath))
+        ssh.run_command("rm -rf {}".format(vmdb_path.join(
+            "/product/menubar/custom_redhat_courses.yml").strpath))
+        ssh.run_command("rm -rf {}".format(vmdb_path.join(
+            "/db/fixtures/miq_product_features/redhat-miq_product_features.yml").strpath))
+        # Restarting the evmserverd to reflect the changes
+        store.current_appliance.restart_evm_service(rude=True)
+    sel.force_navigate("dashboard")
+
+
+def menu_plugin_assert():
+    sel.browser().switch_to_frame(sel.browser().find_element_by_tag_name("iframe"))
+    assert((sel.browser().find_element_by_xpath("//div[@id='legal']/div/div")),
+        "Copyright Information not found")
+    sel.browser().switch_to_default_content()
+    sel.force_navigate('dashboard')
```

Comment:
Comment please on why this is needed....I know it is, but let's tell everyone else so they don't' try and optimize it out.